### ISSUE TITLE
Remove-LabVM throws error: "The term 'Get-FormattedMessage' is not recognized as the name of a cmdlet"

### ISSUE THREAD

**Issue by csandfeld** [VirtualEngine/Lability#85]
In version 0.9.8 Remove-LabVM throws an error

```PowerShell
PS C:\> Remove-LabVM -Name WIN10-10586-3 -Verbose
Get-FormattedMessage : The term 'Get-FormattedMessage' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling o
f the name, or if a path was included, verify that the path is correct and try again.
At C:\Program Files\WindowsPowerShell\Modules\Lability\Src\LabVM.ps1:672 char:38
+             $verboseProcessMessage = Get-FormattedMessage -Message ($ ...
+                                      ~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Get-FormattedMessage:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```
Appear to be a simple typo. I will commit a fix and submit a PR in a moment.

**Comment by iainbrighton** [comment]
Argh balls. How did something so simple get through? Are there no tests for Remove-LabVM?! Would you mind adding a test to catch a future recursion? Thanks!

**Comment by csandfeld** [comment]
I have very limited Pester experience, but guess it will make for a good learning experience. I'll try to come up with something :smile:

**Comment by iainbrighton** [comment]
It will make good experience ;)

You should "just" be able to call Remove-LabVM to catch this. If the test passes and you revert your change - and the test fails - task accomplished!

That's assuming there's no test coverage at present? If you need a hand, let me know. Thanks again for your help.

**Comment by csandfeld** [comment]
Have added pester tests now for Remove-LabVM, and for New-LabVM as well. Currently I have specified a valid MediaId in the test for New-LabVM, but considering that the media ID's might change at some point, its better to mock that part as well. I will look into that next.

**Comment by csandfeld** [comment]
Added mocking of Get-LabMedia to pester tests for New-LabVM.
Let me know if you are OK with the added tests.

**Comment by csandfeld** [comment]
Closed by pull request #86
